src/javascript/home/header.js

```diff
@@ -0,0 +1,24 @@
+function header() {
```

Comment:
It uses ES6 syntax ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png)).
Let's make use of the arrow functions